MODULE.bazel

```diff
@@ -16,7 +16,7 @@ bazel_dep(name = "buildifier_prebuilt", version = "6.4.0", dev_dependency = True
 # until go.work support is released
 git_override(
     module_name = "gazelle",
-    commit = "7d10bf7befce477f518056cfc502939000bea3fa",
+    commit = "476a9447de621f3d6ab0154cc5683b989c79f9c1",
```

Comment:
We may be able to switch to https://github.com/bazelbuild/bazel-gazelle/releases/tag/v0.37.0 by now, I'll follow up on that.